src/helper/batchTransfer.ts

```diff
@@ -19,14 +22,20 @@ const makeVestedConfig = (chain: ChainType, reward: BigNumber): VestingConfig =>
       // 1 month = 403200 blocks
       return {
         srcAddress: 'aXNWfAMUV3YjRoGgceJJpieqzteL4jUWR7LM4xZfHfCGDfQ',
-        perBlock: reward.div(3628800).toFixed(18).toString(),
-        startingBlock: 3628800,
+        perBlock: reward.div(NINE_MONTH_BLOCKS_PER_6_SECONDS),
```

Comment:
I couldn't confirm the hard-coded numbers, but just to clarify, the values assigned here are equivalent to the following logic right?
```
MONTH = 30 * 24 * 60 * 60
BLOCK_SEC = 10 // roughly
VEST_DUR = 10 * MONTH
// reward distribution over 10 months
START_BLOCK = x // Shiden Aug 6th block height
ACCOUNT_TOTAL_REWARD = y // calculated from campaign event
const vestEndBlock = START_BLOCK + (VEST_DUR / BLOCK_SEC)
const vestBlockDur = vestEndBlock - START_BLOCK
const unlockPerBlock = ACCOUNT_TOTAL_REWARD / vestBlockDur
const vestingSchedule = {
  locked: ACCOUNT_TOTAL_REWARD,
  perBlock: unlockPerBlock,
  startingBlock: START_BLOCK
}
```

Reply:
No, I fixed it.  and new comments.